website/.storybook/main.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+
+module.exports = {
+  stories: [
+    "../src/components/**/*.stories.mdx",
+    "../src/components/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@chakra-ui/storybook-addon",
+  ],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-webpack5",
+  },
+  staticDirs: ["../public"],
+  typescript: { reactDocgen: false },
+  webpackFinal: async (config, { configType }) => {
```

Comment:
why is this necessary?

Reply:
` typescript: { reactDocgen: false },` was also necessary for absolute imports... i just cant find the source where i found the solution

Reply:
found [it](https://github.com/storybookjs/storybook/issues/15336#issuecomment-888528747), had an error when trying to use the absolute imports, that was a fix for it..

Reply:
https://github.com/storybookjs/storybook/issues/15336#issuecomment-888528747

Reply:
okay, so the reason is absolute imports, maybe you could document that.

In the link above some users reported added webpack v5 to dev dependencies and everything just worked, however, it pains me that we have to do this.

The offical package has not been updated in a while, which is a bad sign: https://www.npmjs.com/package/@storybook/react-docgen-typescript-plugin


We lose react-docgen, but I think that is ok.

For the webpack final, can you please make it explicit that it is only resolving aliases, and not replacing modules.

```js

    config.resolve.alias = {
      ...config.resolve.alias,
      src: path.resolve(__dirname, "../src"),
    };
    return config;
```